src/OpenFeature.Contrib.Providers.Flagd/InProcessResolver.cs

```diff
@@ -0,0 +1,276 @@
+using System.Net.Http;
+using System.Threading.Tasks;
+using OpenFeature.Model;
+using OpenFeature.Flagd.Grpc.Sync;
+using System;
+using System.IO;
+using System.Linq;
+using System.Net.Security;
+using System.Security.Cryptography.X509Certificates;
+using Grpc.Net.Client;
+using OpenFeature.Flagd.Grpc;
+using System.Net.Sockets;
+using System.Threading;
+using Grpc.Core;
+using Value = OpenFeature.Model.Value;
+
+namespace OpenFeature.Contrib.Providers.Flagd
```

Comment:
minor - may be it's good to namespace in-process vs rpc so that they are proper access modifiers. In the long run this should help in organizing, especially with upcoming json evaluators, offline mode file readers sort of logic.